web/layout_attributes/nesting_flex_layouts/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <link rel="import" href="my_element.html">
+    <script src="packages/browser/dart.js"></script>
```

Comment:
I think this needs to come after the dart script tag, although I am not 100% sure on that